src/env.cc

```diff
@@ -24,6 +25,11 @@ using v8::StackTrace;
 using v8::String;
 using v8::Value;
 
+const int kNodeContextTag = 0x6e6f64;
+void *kNodeContextTagPtr = const_cast<void *>(
+  reinterpret_cast<const void *>(&kNodeContextTag)
+);
```

Comment:
Style nit: Left-leaning pointers (i.e. `void*` rather than `void *`). Also, I think the inner `reinterpret_cast` could just be a `static_cast`?